shakedown/dcos/__init__.py

```diff
@@ -129,6 +131,20 @@ def authenticate_oauth(oauth_token):
         return None
 
 
+def attach_existing_cluster(url):
+    """Attach an existing (already setup) cluster.
+    return: cluster object, or None
+    """
+    clusters = [c.dict() for c in dcos.cluster.get_clusters()]
+    for c in clusters:
+        if url == c['url']:
+            try:
+                cluster.set_attached(dcos.cluster.get_cluster(c['name']).get_cluster_path())
+                return cluster
+            except:
+                return None
+
```

Comment:
I find myself questioning this function ...  looking at `cli/main.py`
the calling function only uses a true/false... Something / None may pass truthiness but it demands what the purposes of this is.   What I question is that the calling function variable `cluster` is True, False, None or Something.   It seems it could be simplified to True/False in this function unless there is some intention that isn't documented or coded.

It appears that the only time this function would be called is if `dcos_url` is passed...  otherwise https://github.com/dcos/shakedown/blob/master/shakedown/cli/main.py#L45 will short circuit.